module-types/Datacom/PSU-120-DC_OLT-DM4610.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 120 DC
+part_number: 800.5080.xx
```

Comment:
Just want to double check, is this the right part number? Can you like to the product/datasheet listing it?

Reply:
According to [this datasheet](https://www.datacom.com.br/uploads/product/file/a4d6b18b9b0cc9b4ac5dccfa22678fdd.pdf), this is the correct definition